### PR TITLE
docs: update guidelines to mention minutes instead of words

### PR DESCRIPTION
**Pull request by bonniepeters**
This file previously suggested that the code of conduct has 196 words in it when there are in fact 197

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
When I check, it is 195 words.

**Comment by bonniepeters** [comment]
@RandellDawson I think the difference between our counts is based on the hyphenated words. Technically, I believe those each count as two separate words. There are 3 hyphenated words so the count would either be 197 (counting each word in a hyphen) or 194 (counting hyphens as 1).

**Comment by RandellDawson** [comment]
Not according to my word processor (MS Word).  It counts hyphenated words as single words.  I think we let @QuincyLarson decide.

**Comment by thecodingaviator** [comment]
Afaik, hyphenated words are one

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Hello. 👋
 
 freeCodeCamp.org is possible, thanks to thousands of kind volunteers like you. We are grateful for your contributions and are excited to welcome you aboard.
 
-We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It's only 196 words long.
+We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It's only 197 words long.
```
Comment (raisedadead) [inline CONTRIBUTING.md]:
```suggestion
We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of-conduct). Please take a moment to read it. It takes only two minutes to read.
```

**Review by raisedadead — DISMISSED** [review]
🤷‍♂

File: CONTRIBUTING.md (hunk 1 above)
Comment (RandellDawson) [reply CONTRIBUTING.md]: So we are treating hyphenated words as two words?

Comment (lynxlynxlynx) [reply CONTRIBUTING.md]: It doesn't matter.

Comment (RandellDawson) [reply CONTRIBUTING.md]: It think it is if we are going through the trouble of even mentioning how many words there are in the COC.

Comment (raisedadead) [reply CONTRIBUTING.md]: Please see the suggested changes.

**Comment by bonniepeters** [comment]
Sorry, I wasn't meaning to waste anyone's time. I am new to the dev community and was trying to participate in open source. @raisedadead I like your suggestion to make it more general so that it wouldn't need nitpicking or updating in the future. Being that I'm new to all this, I'm not entirely sure how to accept your recommended changes... Would you be able to advise on how to do that?

**Comment by lynxlynxlynx** [comment]
Click on the "commit suggestion" button under his proposed change.

**Review by lynxlynxlynx — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Thanks for the PR @bonniepeters!

And don't worry, you didn't waste anyone's time.  It's just that these kinds of issues invite a little too much discussion.  Totally not your fault.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**